yt/yt/server/master/security_server/access_log.cpp

```diff
@@ -278,6 +280,23 @@ bool IsAccessLoggedType(const EObjectType type)
     return typesForAccessLog.contains(type);
 }
 
+bool IsAccessLogEnabled(NCellMaster::TBootstrap* bootstrap) {
+    if (!bootstrap->GetConfigManager()->GetConfig()->SecurityManager->EnableAccessLog) {
+        return false;
+    }
+
+    if (!bootstrap->GetHydraFacade()->GetHydraManager()->IsLeader() &&
+        !bootstrap->GetHydraFacade()->GetHydraManager()->IsRecovery()) {
```

Comment:
And this opening bracket should also be on the following line, as it is a multi-line if condition. The rest of opening brackets in this function are fine.